xchange-bitmex/src/main/java/info/bitrich/xchangestream/bitmex/BitmexStreamingTradeService.java

```diff
@@ -0,0 +1,43 @@
+package info.bitrich.xchangestream.bitmex;
+
+
+import info.bitrich.xchangestream.bitmex.dto.BitmexOrder;
+import info.bitrich.xchangestream.core.StreamingTradeService;
+import io.reactivex.Observable;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.Order;
+import org.knowm.xchange.exceptions.NotYetImplementedForExchangeException;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+
+/**
+ * Created by Declan
+ */
+public class BitmexStreamingTradeService implements StreamingTradeService {
+
+    private final BitmexStreamingService streamingService;
+
+    public BitmexStreamingTradeService(BitmexStreamingService streamingService) {
+        this.streamingService = streamingService;
+    }
+
+    @Override
+    public Observable<Order> getOrders(CurrencyPair currencyPair, Object... args) {
+        String channelName = "order";
+        String instrument = currencyPair.base.toString() + currencyPair.counter.toString();
+        return streamingService.subscribeBitmexChannel(channelName).flatMapIterable(s -> {
+            BitmexOrder[] bitmexOrders = s.toBitmexOrders();
+            return Arrays.stream(bitmexOrders)
+                    .filter(bitmexOrder -> bitmexOrder.getSymbol().equals(instrument))
+                    .filter(BitmexOrder::isNotWorkingIndicator)
+                    .map(BitmexOrder::toOrder).collect(Collectors.toList());
+        });
+    }
+
+    @Override
+    public void submitOrder(Order order, CurrencyPair var1, Object... var2) {
```

Comment:
Remove. This is not the job of xchange-stream.

Reply:
Can we agree that we just remove the interface?
info.bitrich.xchangestream.core.StreamingTradeService

I think its still fine for individual Exchange implementations to have extra exchange specific functionalities. Like getRawFunding() above.

Reply:
Absolutely, there's no problem with providing those extra methods.

I only have a problem with `submitOrder` because there is already a perfectly good way of submitting orders on XChange's `TradeService`; I don't see a good reason to add one here.  `StreamingXXXService`s are meant to just provide streams.

However, the interface is good - it matches the work-in-progress on #274. If the remaining method can just be renamed `getOrderUpdates` then the two will merge cleanly (which reminds me, I really have to get that finished and merged!)

Reply:
I will leave the interface design for you guys first and see if I can match the interface with this implementation. So I remove it anyway at this point.